weldx/tags/core/transformations/local_coordinate_system.py

```diff
@@ -57,6 +60,10 @@ def from_yaml_tree(self, node: dict, tag: str, ctx):
         coordinates = node.get("coordinates")
         if coordinates is not None and not isinstance(coordinates, TimeSeries):
             coordinates = node["coordinates"].data
+            # workaround until the single_pass_weld.wx file needed it the tutorials is
+            # adjusted
+            if not isinstance(coordinates, pint.Quantity):
+                coordinates = Q_(coordinates, "mm")
```

Comment:
if possible we should use something like `_DEFAULT_LEN_UNIT` everywhere in the codebase

Reply:
True, but since this workaround is hopefully going to disappear soon, it shouldn't matter that much ;)